lib/read.js

```diff
@@ -335,6 +356,13 @@ class ReadSql extends AdapterRead {
                     readEnd: this.timeField ? to : new JuttleMoment(Infinity)
                 };
             }
+            
+            if (this.sortFields) {
```

Comment:
@davidvgalbraith you're absolutely right - the pagination was broken and untested. Fixed now. Take another look?